org.opent2t.sample.lamp.superpopular/com.hue.bulb/js/manifest.xml

```diff
@@ -5,10 +5,14 @@
   <!-- associated schemas -->
   <schemas>
     <schema id="org.opent2t.sample.lamp.superpopular" main="true" type="OCF" />
+    <schema id="opent2t.platform" type="OCF" />
     <schema id="oic.core" type="OCF" />
     <schema id="oic.baseresource" type="OCF" />
+    <schema id="oic.oic-link" type="OCF" />
     <schema id="oic.r.switch.binary" type="OCF" />
     <schema id="oic.r.light.dimming" type="OCF" />
     <schema id="oic.r.colour.rgb" type="OCF" />
+    <schema id="oic.r.colour.chroma" type="OCF" />
```

Comment:
I see- thanks Dan-- so then let's just stick with OIC variant which is the British variant.. 

---
In reply to: [94480377](https://github.com/openT2T/translators/pull/143#discussion_r94480377) [](ancestors = 94480377,94472847)